src/workerd/server/workerd.capnp

```diff
@@ -41,7 +41,6 @@ $Cxx.allowCancellation;
 # Any capnp files imported here must be:
 # 1. embedded into workerd-meta.capnp
 # 2. added to `tryImportBulitin` in workerd.c++ (grep for '"/workerd/workerd.capnp"').
```

Comment:
Maybe this comment should go above `using Cxx = import "/capnp/c++.capnp` so that it's next to an import?